config/rewrites.d/developer.rackspace.com.json

```diff
@@ -78,16 +78,11 @@
         },
         {
             "from": "^\\/docs\\/cloud-monitoring\\/?(.*)$",
-            "to": "/docs/rackspace-monitoring/$1",
-            "rewrite": false,
-            "status": 301
-        },
-        {
-            "from": "^\\/how-to\\/cloud-monitoring\\/?(.*)$",
-            "to": "/how-to/rackspace-monitoring/$1",
+            "to": "/docs/#rackspace-monitoring/$1",
```

Comment:
I don't think this substitution will do what you want it to. The fragment (the bit after `#`) needs to be after the final path component of the URL. [Try it in your browser](https://developer.rackspace.com/docs/#rackspace-monitoring/); there's no anchor called "#rackspace-monitoring/", so it's ignored.

You probably just want to drop the `$1` capture and send everything to:

```
"to": "/docs/#rackspace-monitoring"
```